src/SFA.DAS.Roatp.CourseManagement.Web.UnitTests/Models/Standards/StandardDetailsViewModelTests.cs

```diff
@@ -41,6 +50,159 @@ public void ImplicitOperator_ConvertsFromStandardDetails(string regulatorName, b
             viewModel.RegulatorName.Should().Be(regulatorName);
             viewModel.IsStandardRegulated.Should().Be(isRegulated);
             viewModel.CourseDisplayName.Should().Be(expectedCourseDisplayName);
+            viewModel.StandardInfoUrl.Should().Be(standardInfoUrl);
+            viewModel.ContactUsEmail.Should().Be(contactUsEmail);
+            viewModel.ContactUsPhoneNumber.Should().Be(contactUsPhoneNumber);
+            viewModel.ContactUsPageUrl.Should().Be(contactUsPageUrl);
+        }
+
+        [Test]
+        public void ImplicitOperator_ConvertsCourseLocationsFromStandardDetails()
```

Comment:
This is missing tests for Regions and LocationSummary which is calculated in the model. 

If you are adding the above coverage, I suggest you use one test class per method/property.

Reply:
LocationSummary coverage was there, regions coverage added